nautobot/core/models/fields.py

```diff
@@ -126,7 +126,7 @@ class MyModel(models.Model):
             def slugify_function(self, content):
                 return content.replace('_', '-').lower()
 
-            title = models.CharField(max_length=42)
+            title = models.CharField(max_length=255)
             slug = AutoSlugField(populate_from='title')
 
```

Comment:
Ideally we'd change the default `max_length` for `AutoSlugField` (line 136 below and docstring line 86 above) to 255 as well, but need to be careful here as that may result in Apps using this field getting migrations unexpectedly. Maybe better to leave alone?